src/logstash/pipeline/sync_facilities.conf

```diff
@@ -66,9 +66,17 @@ filter {
 }
 output {
   opensearch {
-    hosts => ["http://opensearch-single-node:9200"]
+    hosts => ["${OPENSEARCH_ENDPOINT}"]
+    auth_type => {
+      type => "aws_iam"
+      region => "${AWS_REGION}"
+    }
+    ssl => true
+    ssl_certificate_verification => true
```

Comment:
Does it make any sense since no credentials have been passed?

Reply:
Logstash is authorized in OpenSearch using the IAM role of the task definition. The type of authorization is indicated here.

Reply:
Thanks!

@roninzp do we need SSL?

Reply:
I'd rather keep it since we use https to connect to the Opensearch VPC endpoint.

Reply:
Yes. OpenSearch requires connecting via the SSL protocol

Reply:
@VadimKovalenkoSNF
Based on @roninzp's comments, it looks like the part I commented on shouldn't be changed at all.

Reply:
I have noticed that Logstash returns error while connecting to OpenSearch. @VadimKovalenkoSNF please revert the changes related to this comment.
https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#logsV2:log-groups/log-group/logOpenSupplyHubDevelopmentAppLogstash/log-events/logstash%2Flogstash%2Ff1326fe9fe4946c893f47edfb47a08d2

Reply:
Need to return
```auth_type => {
+      type => "aws_iam"
+      region => "${AWS_REGION}"
+    }
```